packages/client/src/utils/index.test.tsx

```diff
@@ -61,6 +62,18 @@ describe("util", () => {
     expect(isValidPostalcode(" 1055XD ")).toBe(true);
     expect(isValidPostalcode(" 1055 XD ")).toBe(true);
   });
+  test("transformHousenumberToValid", () => {
+    expect(transformHousenumberToValid("")).toBe("");
+    expect(transformHousenumberToValid("110")).toBe("110");
+    expect(transformHousenumberToValid("1 10")).toBe("1 10");
+    expect(transformHousenumberToValid("19-c")).toBe("19 - C");
+    expect(transformHousenumberToValid("19-c        ")).toBe("19 - C");
+    expect(transformHousenumberToValid("          19-c        ")).toBe(
+      "19 - C"
+    );
+    expect(transformHousenumberToValid("10hl")).toBe("10 H L");
+    expect(transformHousenumberToValid("10h l")).toBe("10 H L");
```

Comment:
Can you add tests for:
- `&`
- `101&`
- `101 &`
- `101 &a`
- `101 &1`

Would like to see what happens with this.